src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -337,6 +337,11 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::visualise( const IECore::I
 	return result;
 }
 
+IECore::CompoundDataPtr StandardLightVisualiser::surfaceTexture( const std::string &type, const IECore::InternedString &attributeName, const IECoreScene::ShaderNetwork *shaderNetwork, const IECore::CompoundObject *attributes ) const
+{
+	return nullptr;
```

Comment:
Only that that passed a string to the shader, rather than the image. Is there some other version where we make the return a `DataPtr` and you can return either `StringData` if it's a texture file name, or `CompoundData` if it's the image data itself?

Reply:
Done in https://github.com/GafferHQ/gaffer/pull/3491/commits/8916d9093f2206136577a8b2bceb1a961d0973c6